core/drivers/imx/mu/imx_mu.c

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2022 NXP
+ */
+#include <assert.h>
+#include <drivers/imx_mu.h>
+#include <kernel/delay.h>
+#include <kernel/spinlock.h>
+#include <string.h>
+#include <trace.h>
+
+#include "imx_mu_platform.h"
+
+#define RX_TIMEOUT (100 * 1000)
+
+static unsigned int s_mu_spinlock = SPINLOCK_UNLOCK;
```

Comment:
What's the story with the `s_` prefix? If it's to some variant of coding attributes into the name of the variable please drop it.

Reply:
Ok, I will remove it